src/locales/cs.json

```diff
@@ -662,20 +659,25 @@
   "app.forkExerciseForm.submit": "Zduplikovat úlohu",
   "app.forkExerciseForm.submitting": "Duplikování ...",
   "app.forkExerciseForm.success": "Úloha zduplikována",
-  "app.generateToken.copied": "Zkopírováno!",
-  "app.generateToken.copyToClipboard": "Kopírovat do schránky",
-  "app.generateToken.day": "1 Den",
-  "app.generateToken.expiration": "Vyprší za:",
-  "app.generateToken.hour": "1 Hodinu",
-  "app.generateToken.lastToken": "Poslední token:",
-  "app.generateToken.month": "1 Měsíc",
-  "app.generateToken.scope.master": "Čtení a zápis",
-  "app.generateToken.scope.readAll": "Pouze čtení",
-  "app.generateToken.scope.refresh": "Povolit nekonečné prodlužování",
-  "app.generateToken.scopes": "Rozsahy platnosti:",
-  "app.generateToken.warnBothMasterAndReadAll": "Povolený rozsah 'pouze pro čtení' nebude mít žádný efekt, neboť byl zároveň vybrán rozsah 'čtení a zápis', který v sobě 'čtení' obsahuje implicitně.",
-  "app.generateToken.week": "1 Týden",
-  "app.generateToken.year": "1 Rok",
+  "app.generateTokenForm.copied": "Zkopírováno!",
+  "app.generateTokenForm.copyToClipboard": "Kopírovat do schránky",
+  "app.generateTokenForm.day": "1 Den",
+  "app.generateTokenForm.expiration": "Vyprší za:",
+  "app.generateTokenForm.hour": "1 Hodinu",
+  "app.generateTokenForm.lastToken": "Poslední token:",
+  "app.generateTokenForm.month": "1 Měsíc",
+  "app.generateTokenForm.scope.master": "Čtení a zápis",
+  "app.generateTokenForm.scope.readAll": "Pouze čtení",
+  "app.generateTokenForm.scope.refresh": "Povolit nekonečné prodlužování",
+  "app.generateTokenForm.scopes": "Rozsahy platnosti:",
+  "app.generateTokenForm.validate.expirationTooLong": "Povolený rozsah 'pro čtení a zápis' má v tuto chvíli omezenou maximální životnost na jeden týden.",
```

Comment:
I don't like the single quotes here ... especially that `pro` word is in the enclosed section.